Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -0,0 +1,72 @@
+---
+versionFrom: 8.0.0
+---
+
+## Non-synchronised/replicated file system
+
+If the file system on your servers isn't performing any file replication then no _Umbraco_ configuration file changes are necessary.
+
+Depending on the configuration and performance of the local storage you might need to consider Examine Directory Factory Options and the Umbraco temporary storage location. 
```

Comment:
environment's local storage, you might need to consider...

then I would link to the docs about 'Examine's Directory Factory Options' and also to the bit above where the Temporary Storage Location is set?